### ISSUE TITLE
Making our own database error

### ISSUE THREAD

**Issue by daniel251994** [tseemann/abricate#153]
I have followed the steps exactly as described for making our own database using Abricate. In my case, i am making a database of arcobacter that contains 46 sequences, and the database name assigned is "arcodb" while the sequence file in fasta format is named as "sequences.fa". However after this step : makeblastdb -in sequences.fa -title arcodb -dbtype nucl -hash_index, the database arcodb is not listed after the "abricate --list" command, even after the database is created successfully. I am currently using abricate through conda. 
I kindly request you to help me with this issue.
![abricate](https://user-images.githubusercontent.com/58763668/95844814-f8fbaa00-0d66-11eb-87e0-e489b5024b76.png)

**Comment by buihoangphuc412** [comment]
I think you should delete the format .fa of sequences.fa into sequences only when make your own databases. Since abricate cannot identify sequences.fa in db.

**Comment by daniel251994** [comment]
Ok i will try that soon, Thank you very much for your reply.

**Comment by daniel251994** [comment]
I did remove the format .fa but i got the same results.
![abricate2](https://user-images.githubusercontent.com/58763668/95846640-4e38bb00-0d69-11eb-9dc3-64b3d4b65539.png)

**Comment by buihoangphuc412** [comment]
I see. I usually do not use the hash_index argument but I think it is no problem. Did you try command?
`abricate --setupdb`
It will add your own databse into the database list!

**Comment by daniel251994** [comment]
Yes, before running the makeblastdb, i already ran abricate --setupdb but arcodb wasn't added so i went for makeblastdb.

**Comment by buihoangphuc412** [comment]
After you makeblastdb, you should type this command, not before!

**Comment by daniel251994** [comment]
ok i will check . Thank you once again

**Comment by daniel251994** [comment]
It still didn't work. But I was able to avoid this issue by not using conda, in other words I have install abricate separately from source outside conda and everything went fine.

**Comment by tseemann** [comment]
The DB needs to be in the `--datadir` folder.
